server/server.js

```diff
@@ -0,0 +1,30 @@
+// Server
```

Comment:
The server is hosted on port 5000. When I launch the client webpage using http-server, this is hosted on port 8080.
However, on both port the DOM is identical (see screenshot), whereas I would not expect a styled page for the API server.
![image](https://user-images.githubusercontent.com/88339527/151336653-a75a8475-8ccd-4a2a-8dda-26582e9c033b.png).